src/services/bundle_repository.js

```diff
@@ -78,14 +80,33 @@ export default class BundleRepository {
     return await downloadJSONFromGridFSBucket(bundleId, this.bundlesBucket);
   }
 
-  async createBundleMetadata(bundleId, storagePeriods, status = BundleStatusStates.unknown) {
+  async createBundleMetadata(bundleId, storagePeriods, status = BundleStatuses.unknown, additionalRepositoryFields = {}) {
```

Comment:
Looks like `additionalRepositoryFields` is not used anywhere but in test and can be deleted.

Reply:
Leftovers from adding holdUntil field :)